inc/thorvg_capi.h

```diff
@@ -126,6 +126,7 @@ TVG_EXPORT Tvg_Result tvg_paint_transform(Tvg_Paint* paint, const Tvg_Matrix* m)
 /* Shape API                                                            */
 /************************************************************************/
 TVG_EXPORT Tvg_Paint* tvg_shape_new();
+TVG_EXPORT Tvg_Paint* tvg_shape_duplicate(Tvg_Paint* paint);
```

Comment:
Please change api to  => tvg_paint_duplicate(const Tvg_Paint* paint);

Right now, we assume this paint is shape, 
but we must support all paints types for duplicate()

Reply:
done